ui/raidboss/data/07-dt/dungeon/ihuykatumu.ts

```diff
@@ -0,0 +1,193 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'Ihuykatumu',
+  zoneId: ZoneId.Ihuykatumu,
+  timelineFile: 'ihuykatumu.txt',
+  triggers: [
+    // ** Prime Punutiy ** //
+    {
+      id: 'Ihuykatumu Prime Punutiy Punutiy Press',
+      type: 'StartsUsing',
+      netRegex: { id: '8E8C', source: 'Prime Punutiy', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Decay',
+      type: 'StartsUsing',
+      netRegex: { id: '8E99', source: 'Ihuykatumu Flytrap', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get under Flytrap',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Inhale',
+      type: 'StartsUsing',
+      netRegex: { id: '8E8E', source: 'Prime Punutiy', capture: false },
+      durationSeconds: 10, // prolonged ground damage
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Stay out of inhale',
```

Comment:
Would this be better as a `Sides` text instead?

Reply:
> Would this be better as a `Sides` text instead?

It's actually a fairly narrow conal (as compared to Drowsie's `Sneeze`, e.g.), but it's a persistent ground AoE for 5s that will apply damage every second if the player is standing in it after the initial telegraph/hit.  You don't really need to be at the sides of the boss to avoid, and the boss in on the wall anyway.  I'm not sure though if there's a better pre-canned response we've used for persistent ground effects that would be appropriate though?

Reply:
I was more concerned with the callout text being long/distracting mid-pull, as opposed to most default trigger texts that are concise.